examples/webgl_morphtargets_horse.html

```diff
@@ -69,11 +71,11 @@
 
 				//
 
-				const light1 = new THREE.DirectionalLight( 0xefefff, 1.5 );
+				const light1 = new THREE.DirectionalLight( 0xefefff, 5 );
 				light1.position.set( 1, 1, 1 ).normalize();
 				scene.add( light1 );
 
-				const light2 = new THREE.DirectionalLight( 0xffefef, 1.5 );
+				const light2 = new THREE.DirectionalLight( 0xffefef, 5 );
```

Comment:
>light2.position.set( - 1, - 1, - 1 ).normalize();

This is an old example, but note that It is not typical, in a physical environment, to have directional lights shining from the ground upward. Of course, it can happen.

It is also not typical to have a horse that is 300 meters in height.

So I agree with @Mugen87 in this sense: it is important to be explicit in stating our objectives prior to updating the remaining examples. I am not sure our objectives are clear at this point.

Thank you, @donmccurdy, for starting this process.

Reply:
>I am not sure our objectives are clear at this point.

They are not, as far as I can tell.

Reply:
Yes. How about we reconsider our objectives in https://github.com/mrdoob/three.js/pull/25479 before updating more examples?

Reply:
Continued in https://github.com/mrdoob/three.js/pull/25479#issuecomment-1482988818.